source/test/ssmplaybook.test.ts

```diff
@@ -88,36 +81,29 @@ function getSsmRemediationRunbook(): Stack {
 }
 
 test('Test Shared Remediation Generation', () => {
```

Comment:
minor nitpick -- could this name be more descriptive? (i.e. "xyz stack includes shared remediation")

Reply:
Probably could be better, but see above, that's not the focus of this change